lib/useBrand.ts

```diff
@@ -1,11 +1,24 @@
-import { getBrand } from "@sirclo/nexus";
-import { GRAPHQL_URI } from '../components/Constants';
-import { IncomingMessage } from 'http';
+import { ParsedUrlQuery } from 'querystring'
+import { GRAPHQL_URI } from '../components/Constants'
```

Comment:
```suggestion
import { GRAPHQL_URI } from 'components/Constants'
```

dibikin begidu bisa tydac ngab?